tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1239,6 +1239,19 @@ a dictionary can also be used with a named enum.
         # prints '["STATE_IDLE", "STATE_JUMP", "STATE_SHOOT"]'
         print(State.keys())
 
+And since the keys are inside a dictionary, if you wish to iterate the values assigned to those enums, you must iterate the values of the enums
```

Comment:
```suggestion
And since the keys are inside a dictionary, if you wish to iterate over the values assigned to those enums, you can use the :ref:`Dictionary.values() <class_Dictionary_method_values>` method.
```